doc/config.md

```diff
@@ -0,0 +1,129 @@
+# Configuration format of qem-bot
+
+Configuration, in another way also called metadata of qem-bot, is held in yaml files with one file per "product". Additionally, there is one special config file called `singlearch.yml` which contains a list of packages that exist only on a single architecture.
+Configuration usually spans over multiple files. And default location excepted from qem-bot is `/etc/openqabot/`
+
+```bash
+/etc/openqabot ❯❯❯ tree                   
+.                          
+├── singlearch.yml               
+├── leap154.yml                   
+├── sles15sp3.yml        
+├── micro53.yml          
+...
+```
+Example of `singlearch.yaml`:
```

Comment:
This says `.yaml` and the two mentions above say `.yml`. Do both extensions work? Then it's probably worth mentioning that as well.

Reply:
This was marked as resolved, but i don't see anything in the docs about it.

Reply:
ahh, forgot commit. Thx for spotting